From fda28bbd6718bf113061292d8cec15084060c923 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Wed, 19 Jul 2023 15:21:46 -0700
Subject: [PATCH] Disable on_frame_base_surface wl_surface_commit

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 shell/wayland/window.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/wayland/window.cc b/shell/wayland/window.cc
index c74b438..f244f08 100644
--- a/shell/wayland/window.cc
+++ b/shell/wayland/window.cc
@@ -257,7 +257,7 @@ void WaylandWindow::on_frame_base_surface(void* data,
   window->m_fps_counter++;
   window->m_fps_counter++;
 
-  wl_surface_commit(window->m_base_surface);
+  //wl_surface_commit(window->m_base_surface);
 }
 
 uint32_t WaylandWindow::GetFpsCounter() {
-- 
2.41.0

